.travis.yml

```
@@ -20,6 +20,7 @@ addons:
 env:
   - CPPFLAGS_ADD='' FCFLAGS_REAL_KIND='-fdefault-real-8 -fdefault-double-8'
   - CPPFLAGS_ADD='-DOVERLOAD_R4 -DOVERLOAD_R8' FCFLAGS_REAL_KIND=''
+  - CPPFLAGS_ADD='' FCFLAGS_REAL_KIND='-fdefault-real-8 -fdefault-double-8 -fopenmp'
```

I would rather see a new OPENMP environment

```suggestion
  - CPPFLAGS_ADD='' FCFLAGS_REAL_KIND='-fdefault-real-8 -fdefault-double-8' OPENMP='-fopenmp'
```

You'll then need to add `$OPENMP` to the `FCFLAGS` on line 33.  (also add to tother environments.